### PR TITLE
Added configuration files to onboard new users

### PR DESCRIPTION
**Pull request by wpoehlm**
To onboard new users, I've added configuration files for Sophia Jobe, Verena Chung, Jared Hendrickson, and Wei-An Chen.

### CODE REVIEW

**Comment by wpoehlm** [comment]
@BrunoGrandePhD please note that IT has not yet added them to the Jumpcloud groups, so if this is necessary before merging please hold off until I comment again.

**Review by thomasyu888 — APPROVED** [review]
LGTM - will let Bruno merge when he's back

File: config/projects-prod/verena-chung-project.yaml
Comment (vpchung) [reply config/projects-prod/verena-chung-project.yaml:10]: @wpoehlm yes, those are correct - thanks!

**Comment by wpoehlm** [comment]
@BrunoGrandePhD users have been added to the JumpCloud groups, so this should be ready to merge pending your approval

File: config/projects-prod/sophia-jobe-project.yaml
Comment (BrunoGrandePhD) [inline config/projects-prod/sophia-jobe-project.yaml:22]: Do you want to add yourself to these `projects-prod` stacks so you can help debug issues during the workshop?

**Review by BrunoGrandePhD — CHANGES_REQUESTED** [review]
I'm appr

File: config/projects-prod/sophia-jobe-project.yaml
Comment (wpoehlm) [reply config/projects-prod/sophia-jobe-project.yaml:22]: I like this idea theoretically, but if these projects are going to be potentially used for real data in the future then it might not be appropriate for me to have direct access depending on the project?

Comment (BrunoGrandePhD) [reply config/projects-prod/sophia-jobe-project.yaml:22]: I don't think we should have Tower projects named after individuals. We don't have enough AWS Batch slots for compute environments to support this approach. Once the workshop is done, we should decommission these individual-based projects in favor of grant-based projects. We don't have enough time to identify the latter, so we can stick with the former just for tomorrow.

Comment (BrunoGrandePhD) [reply config/projects-prod/sophia-jobe-project.yaml:22]: You can specify that no PHI data should be used so you can provide support during the workshop.

Comment (wpoehlm) [reply config/projects-prod/sophia-jobe-project.yaml:22]: In the case of Jared and Wei-An, I'd like these to be long term projects they can use. Would you recommend that I remove the individual accounts and add them to mine, or can we keep those as is (and still use them after the workshop)?

Comment (BrunoGrandePhD) [reply config/projects-prod/sophia-jobe-project.yaml:22]: It depends on how you want to organize the workflow runs. I'm less concerned about running into the AWS Batch limit on the STRIDES account since it has very few Tower projects so far. If there will be overlap in the data used, then a single Tower project might make more sense so you don't have to duplicate data across S3 buckets.

**Comment by wpoehlm** [comment]
I've given myself read-write access to Verena and Sophia's project for the workshop. I would prefer to keep Jared and Wei-An's projects separate, and I have read/write access on them as well, so if you're not worried about batch limits on strides then we can leave it?

**Review by BrunoGrandePhD — APPROVED** [review]

**Pull request merged**
